### ISSUE TITLE
Add a new action to just select prev and next panes

### ISSUE THREAD

**Issue by diwasrimal** [wez/wezterm#6424]
**Is your feature request related to a problem? Please describe.**
Not exactly

**Describe the solution you'd like**
Creating new action(s) that lets people navigate through prev and next panes. This will allow people to just bind  two keys to navigate through panes, instead of mapping four different ones. Most people like me with small screens will just use two or three panes at max. So an action to just cycle next/prev panes seems like a good feature to have.

**Describe alternatives you've considered**
I don't know of any

**Comment by tlucanti** [comment]
Yes, this will be very helpful. I think it will be enough just to add two new `ActivatePaneDirection`s: `Clockwise` and `Counterclockwise`. Or just `Next` and `Prev` (sorting by creation order).

**Comment by mgpinf** [comment]
You can use `act.ActivatePaneDirection 'Prev'` and `act.ActivatePaneDirection 'Next'` to achieve the desired result.
